test/python/circuit/test_gate_power.py

```diff
@@ -186,7 +186,7 @@ def test_invariant1_int(self, n):
     @data(-3, -2, -1, 1, 2, 3)
     def test_invariant2(self, n):
         """Test op^(n) * op^(-n) == I"""
-        result = Operator(SGate()).power(n) & Operator(SGate()).power(-n)
+        result = Operator(SGate()) ** n & Operator(SGate()) ** -n
```

Comment:
This change is really something quite different.

Reply:
I've reverted the tests and just made a separate one.